extensions/fairifier-rdf-extension/dtls/fairifier/FileSystemRdfSkeletonImp.java

```diff
@@ -0,0 +1,92 @@
+package org.dtls.fairifier;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class FileSystemRdfSkeletonImp implements RdfSkeletonService{
+    private static final Path SAVELOCATION = Paths.get(System.getProperty("user.home") + "/.local/share/openrefine/");
```

Comment:
Is there an existing storage location available through openrefine? The location where the projects themselves are persisted maybe?

Reply:
That would be this location, but maybe i could also pull it from the configs